build.ps1

```diff
@@ -137,7 +137,7 @@ $appCommonPackageVersion = "3.0.0"
 Import-Module -Name "$PSScriptRoot/eng/build-helpers.psm1" -Force
 Import-Module -Name "$PSScriptRoot/eng/package-manager.psm1" -Force
 Import-Module -Name "$PSScriptRoot/eng/database-manager.psm1" -Force
-function Clean {
+function DotNetClean {
```

Comment:
PowerShell 7 does not like having a function called `Clean`. I couldn't figure out why, so I just renamed it.